Shoko.TrayService/Shoko.TrayService.csproj

```diff
@@ -15,6 +15,8 @@
         <Configurations>Debug;Release;ApiLogging</Configurations>
         <ApplicationIcon>..\Shoko.Server\db.ico</ApplicationIcon>
         <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
+        <NoWarn>CA1416</NoWarn>
```

Comment:
Would this be better to have only specific instances of this listed and ignored if required? Removing it on the whole project to me seems a little excessive.

Reply:
`CA1416: Validate platform compatibility` simply won't make any sense, in a windows only project.

Reply:
I am aware on what it is, but, that still allows you to validate that you're not using Linux/OSX apis on a, windows-only project. There's two sides to the sword here.